bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -120,9 +122,6 @@ public Response doCreate( final String packageType, final String type, final Str
                 builderModifier.accept( builder );
             }
             response = builder.build();
-
-            // generating .http-metadata.json for PUT request to resolve some header requirements
-            contentController.generateHttpMetadataHeaders( transfer, request, response );
```

Comment:
So this means maven uploads won't generate this any more, right? I wish we could leave that in place. It's a separate JIRA we've had in the backlog for a long time now.

But I'm not sure if that will cause problems for what you're doing.

Reply:
Yes, you are right, @jdcasey , I move to npm itself, not for whole packagetypes now.

In the case of npm, if generates the http-metadata.json for uploads, it seems the meta will not be replaced when downloads, while this kind of http-meatadata is applicable for npm, I’m not sure what’s the maven needs for downloads, whether it’s a strict requirement, I recommend to add this after 1.2 and we could have some tests to verify it, to reduce some risk of boundaries we haven’t touch yet？

Reply:
:+1: